stylesheets/modules/b2b-account/orders.less

```diff
@@ -6,7 +6,24 @@
           margin: 0 @gutter/4 0 0;
       }
    }
-   
+
+   #searchName
```

Comment:
Move all of these styles to quotes.less file. since these are not related to orders

Reply:
done